ui/app/controllers/clients/client/index.js

```diff
@@ -27,11 +31,19 @@ export default class ClientController extends Controller.extend(Sortable, Search
     {
       onlyPreemptions: 'preemptions',
     },
+    {
+      qpStatus: 'status',
+    },
+    {
+      qpTaskGroup: 'taskGroup',
```

Comment:
For this view, since it shows all allocations in a client, I think I would like to see a `Job` filter instead of `Task Group`. 

Allocations for the same task group have an inherent anti-affinity, meaning it's unlikely that there would be more than one allocation for the same task group in the same client. 

There's also the case where different jobs have the same task group name, but I don't think that a filter would be useful in this scenario since task groups from different jobs are not related.

Another filter that I think would be useful here is namespace, but we don't display in in the table (though we probably should 😄)

Reply:
Lets circulate this back to @juliezzhou and @mikenomitch .

Reply:
That all sounds good to me.

The namespace makes sense to me too (only in the clients view to be clear). Also since we don't have the client anymore we have room for it. @ChaiWithJai if the data is handy, I'd throw that in there too but, if its not available, then no worries if we drop it.

Will including namespace be weird for people only using a single namespace though? Just always showing "default" might be noisy with no benefit. @lgfa29 what do you think there?

Reply:
Personally I find that hiding namespaces, specially after they've been open sourced, diminishes their value since it's hard to learn that they even exist. It may be noisy, but I'm not sure if it would be too bad.

Reply:
Yeah fair - it will help with discoverability of NSs too.

So I think the filters we want on the clients page are:
- Status
- Job
- Namespace

Reply:
I like the idea of exploring this and appreciate the feedback, but I’m also kinda wanting to just ship the design we had decided on w/ Julie…maybe we can revisit down the road and iterate on it a bit?